masterdata/returnRequest/schema.json

```diff
@@ -116,7 +116,7 @@
           "imageUrl": { "type": "string" },
           "unitMultiplier": { "type": "number" },
           "sellerId": { "type": "string" },
-          "sellerName": { "type": "string" },
+          "sellerName": { "type": ["string", "null"] },
```

Comment:
We don't need to make it `null` as well. The field is not listed as a required one, so it can be undefined.